### PR TITLE
git-dit: init at 0.1.0

### PR DESCRIPTION
**Pull request by Profpatsch**
A decentral issue tracker for git

- [x] NixOS
- [x] Tested execution of *some* binary files (usually in `./result/bin/`)

cc @neithernut @matthiasbeyer

### CODE REVIEW

**Comment by matthiasbeyer** [comment]
Muahhahaha.

Well, no. I wouldn't package it, actually. It is a proof of concept, not a tool that should be used. But I may be biased here.

**Comment by matthiasbeyer** [comment]
Either way, if it gets into nixpkgs, I would like to be maintainer of the package, too!

**Comment by Profpatsch** [comment]
Sure thing, I added you.

**Comment by Profpatsch** [comment]
> Well, no. I wouldn't package it, actually.

Well, I’d like to use it and there is a release. :P

File: pkgs/applications/version-management/git-and-tools/git-dit/default.nix [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  dontBuild = true;
+
+  postPatch = ''
+    # resolve binaries to the right path
+    sed -e "s|exec git-dit-|exec $out/bin/git-dit-|" -i git-dit
+
+    # we change every git dit command to the local subcommand path
+    # (git dit foo -> /nix/store/…-git-dit/bin/git-dit-foo)
+    for script in git-dit-*; do
+      sed -e "s|git dit |$out/bin/git-dit-|g" -i "$script"
```
Comment (Mic92) [inline pkgs/applications/version-management/git-and-tools/git-dit/default.nix]: Breaks calling `git-dit`:

```
$ git-dit 
/nix/store/jrdv2q1s50ks0hf6pnksi634rp0pmvsx-git-dit-0.1.0/bin/git-dit: line 54: /nix/store/jrdv2q1s50ks0hf6pnksi634rp0pmvsx-git-dit-0.1.0/bin/git-dit-: No such file or directory
```

Comment (Profpatsch) [inline pkgs/applications/version-management/git-and-tools/git-dit/default.nix]: Yeah, that’s a bug in the release.
It’s `for script in git-dit-*`, so `git-dit` isn’t in that loop.

**Pull request merged**